Multisig/UI/Settings/Passcode/SecuritySettingsViewController.swift

```diff
@@ -217,14 +217,15 @@ class SecuritySettingsViewController: UITableViewController {
 
     private func toggleBiometricsSecurityLock(newLockMethod: LockMethod) {
         guard newLockMethod != App.shared.securityCenter.lockMethod else { return }
-        App.shared.securityCenter.changeSecuritySettings(passcode: nil,
-                                                         lockMethod: newLockMethod) { [unowned self] error in
-            if let error = error {
-                LogService.shared.error("Failed to update lock method", error: error)
-            }
-
-            reloadData()
-        }
+        // TODO: biometrics changed!
+//        App.shared.securityCenter.changeSecuritySettings(newPlaintextPasscode: nil,
```

Comment:
needed?

Reply:
yes, this needs rewrite with the new security center approach, so that we could react to the new lock method change